ci-operator/step-registry/install-operators/install-sriov-network-operator/install-operators-install-sriov-network-operator-commands.sh

```diff
@@ -0,0 +1,132 @@
+#!/bin/bash
```

Comment:
Doesn't the `sriov-network-operator` team have this step? If yes, how do they run their test cases in Prow regression tests CI?

Reply:
The sriov-network-operator  had a tag:stage-only, like  https://github.com/openshift/verification-tests/blob/master/features/networking/sriov.feature#L6, seems about operators are all never runned in prow before : https://github.com/openshift/verification-tests/pull/2230/files, because in no stage cluster in prow.